src/infra/QuestionSubmissionServiceImpl.ts

```diff
@@ -54,14 +54,14 @@ export class QuestionSubmissionServiceImpl implements QuestionSubmissionService
         this.endpoint,
         queryParams,
         body,
-        getClientSdk(request.customClientSdk),
+        getClientSdk(request.additionalHttpHeaderValues),
         this.config.token
       )
       : await this.httpService.post<ApiResponse>(
```

Comment:
since we mostly keep get/post methods to be pretty generic, would it make sense to pass something like a headerParams or additionalHeaderParams which includes `Client-SDK` instead of passing it directly in as an argument?